daml-script/dump/src/it/scala/com/daml/script/dump/IT.scala

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.script.dump
+
+import java.io.IOException
+import java.nio.file.{Files, FileVisitResult, Path, SimpleFileVisitor}
+import java.nio.file.attribute.BasicFileAttributes
+import java.util.UUID
+
+import akka.stream.scaladsl.Sink
+import com.daml.bazeltools.BazelRunfiles
+import com.daml.lf.language.Ast.Package
+import com.daml.lf.data.Ref
+import com.daml.lf.data.Ref.PackageId
+import com.daml.lf.engine.script.{GrpcLedgerClient, Participants, Runner, ScriptTimeMode}
+import com.daml.ledger.api.domain
+import com.daml.ledger.api.refinements.ApiTypes.ApplicationId
+import com.daml.ledger.api.testing.utils.{AkkaBeforeAndAfterAll, SuiteResourceManagementAroundEach}
+import com.daml.ledger.api.v1.command_service.SubmitAndWaitRequest
+import com.daml.ledger.api.v1.commands._
+import com.daml.ledger.api.v1.ledger_offset.LedgerOffset
+import com.daml.ledger.api.v1.transaction_filter.{Filters, TransactionFilter}
+import com.daml.ledger.api.v1.{value => api}
+import com.daml.ledger.client.LedgerClient
+import com.daml.ledger.client.configuration.{
+  CommandClientConfiguration,
+  LedgerClientConfiguration,
+  LedgerIdRequirement,
+}
+import com.daml.lf.archive.{Dar, DarReader, Decode}
+import com.daml.platform.sandbox.services.TestCommands
+import com.daml.platform.sandboxnext.SandboxNextFixture
+import scalaz.syntax.tag._
+import scalaz.syntax.traverse._
+import spray.json._
+
+import scala.sys.process._
+import org.scalatest.freespec.AsyncFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+final class IT
```

Comment:
I don’t really like that test setup but I’m not quite sure what we should do instead. I think we probably want a function to diff ledger API transaction trees allowing for alpha-renaming of contract ids and party ids. Then we can make this fairly generic. However, that seems like a larger task so I’d prefer to leave it out of this PR•